source/_components/dominos.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "Dominos Pizza"
+description: "Instructions how to setup Dominos Pizza ordering within Home Assistant."
+date: 2017-11-05 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: dominos.png
+ha_category: Other
+ha_version: 0.57
```

Comment:
`0.57` is already out, just like `0.58`. You might want to update this to `0.59`.